cmd/spire-server/cli/entry/delete.go

```diff
@@ -140,12 +141,20 @@ func (c *deleteCommand) prettyPrintDelete(env *commoncli.Env, results ...any) er
 			result.Status.Message)
 	}
 
+	var summaryMsg string
 	if len(failed) > 0 {
 		env.Printf("\n\n")
+		summaryMsg = fmt.Sprintf("Deleted %d entries successfully, but failed to delete %d entries", len(succeeded), len(failed))
```

Comment:
We can move the variable to be inside the if condition:
```suggestion
if len(failed) > 0 {
env.Printf("\n\n")
		summaryMsg := fmt.Sprintf("Deleted %d entries successfully, but failed to delete %d entries", len(succeeded), len(failed))
```